### PR TITLE
Seo fixes

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Fix schema
- 1. location of schema for projects without venue
- 2. Add eventAttendanceMode
- 3. Add organizer
- 4. remove invalid offer
2. Add profile name to project page title

### CODE REVIEW

File: funnel/templates/project.html.jinja2 [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {% extends "layout.html.jinja2" %}
+{% set title_suffix = project.profile.title %}
```
Comment (jace) [inline funnel/templates/project.html.jinja2:2]: This is not being used. Also, it's usually a prefix.

File: funnel/templates/project.html.jinja2 [hunk 2]
```diff
@@ -17,6 +18,8 @@
       "name": {{ project.title|tojson }},
       "url": {{ project.url_for(_external=true)|tojson }},
       "startDate": {{ project.schedule_start_at_localized|tojson }},
+      "endDate": {{ project.schedule_end_at_localized|tojson }},
+      "eventStatus": "https://schema.org/EventScheduled",
```
Comment (jace) [inline funnel/templates/project.html.jinja2:22]: I think this needs an `if` condition that checks for `schedule_start_at` being null.

File: funnel/templates/project.html.jinja2 (hunk 1 above)
Comment (vidya-ram) [reply funnel/templates/project.html.jinja2:2]: We have defined it in layout
https://github.com/hasgeek/funnel/blob/master/funnel/templates/layout.html.jinja2#L5

File: funnel/templates/project.html.jinja2 (hunk 2 above)
Comment (vidya-ram) [reply funnel/templates/project.html.jinja2:22]: We are checking it on line 13

**Pull request merged**